cypress/support/commands.js

```diff
@@ -23,3 +23,21 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+Cypress.Commands.add('findByPlaceholder', (placeholder) => {
+  cy.get(`[placeholder="${placeholder}"]`);
```

Comment:
The 'findByPlaceholder' command does not return anything. This means that you can't chain other Cypress commands off it. It should return the result of the 'cy.get' command.

Reply:
Fixed